1001-2000/LIT1295Dersan.xml

```diff
@@ -152,160 +141,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                  <ab>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ። ድርሳን፡ ወነገር፡ ዘቅዱስ፡ አ<choice><sic>በ</sic><corr>ባ</corr></choice>፡ 
+                     <persName ref="PRS9523Timothy">ደማቲዎስ፡</persName> 
+                     ሊቀ፡ ጳጳሳት፡ ዘለእስክንድርያ፡ ትነበብ፡ በበዓለ፡ ቅዱሰ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ አመ፡ ፲ወ፪ለህዳር፡ ኣርፋታኤል፡ አዘፋታኢል፡ ሕዝትናኤል፡ ምርሜል፡ ስመ፡ አምላክ፡ ኢየሱስ፡ 
+                     ክርስቶስ፡ ቃል፡ ቀዳማዊ፡ ወልደ፡ እግዚአብሔር፡ ሕያው፡ ወወልደ፡ ማርያም፡ ሥግው፡ ያርኅቅ፡ እም<choice><sic>ኒ</sic><corr>ኔ</corr></choice>ነ፡ 
+                     ኃይለ፡ ሳይግናት፡ ገሙናነ፡ መንፈስ፡ ወያድክም፡ ሀይሎሙ፡ ለመናፍስት፡ 
+                     ርኩሳን፡ ወያርእየነ፡ እግዚአብሔር፡ ክብረ፡ ዓቢየ፡ ዘወሀቦ፡ ለቅዱስ፡ ሚካኤል፡ እም<choice><sic>ኩ</sic><corr>ኵ</corr></choice>ሎሙ፡ ቅዱሳን፡ 
+                     መላእክተ፡ በረክተ፡ ረድኤቱ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ 
+                     <gap reason="omitted"/> ለዓ፡ ዓሜን፡ <gap reason="omitted"/> 
+                     ወኢታንብሮ፡ ውስተ፡ ቢት፡ እኩይ፡ ከመ፡ እይኩን፡ ስላቀ፡ ለ<choice><sic>ስ</sic><corr>ሳ</corr></choice>ይጣን፡ ኢትዝርዎ፡ ለአብዳን፡ ዘአልቦሙ፡ አሚን፡ ወኢጥሙቃን፡ አላ፡ እቀቦ፡ በተአምኖቱ፡ ለመልአካ፡ ምሕረት፡ ብርሃን፡ 
+                     ኦቅዱስ፡ <add resp="CH">ሚካኤል፡</add> ሊቀ፡ መላእክት፡ ባልሖ፡ እመከራ፡ ሥጋ፡ ወነፍስ፡ ለጸሐፌ፡ ዝንቱ፡ 
+                     ድ<add resp="CH">ር</add><choice><sic>ስ</sic><corr>ሳ</corr></choice>ን፡ ወለዘአጽሐፎ፡ <gap reason="omitted"/> 
+                     ዓሜን፡ ወዓሜን፡ እይኩን፡ ለይኩ<add resp="CH">ን</add>፡ </ab>                  
+               </div>
+               
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="IntroductionMiracles">
+                  <label xml:lang="en">Introduction to the miracles performed by Michael</label>
+                  <note>The following text is based on <ref type="mss" corresp="BMLacq776"/>, which is possibly similar to that one in 
+                     <ref type="mss" corresp="BAVet82"/> and <ref type="mss" corresp="BAVet121"/>, but for the latter included to the
+                     section for Maskaram.</note>
+                  <ab>በስመ<supplied reason="omitted">፡ </supplied>አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ። ንቀድም፡ በረዴዓተ፡ እግዚአብሔር፡ ጽሒፈ፡ ተአምሪሁ፡ 
+                     ለቅዱስ፡ <add resp="CH">ሚካኤል፡</add> ሊቀ፡ መላእክት፡ ትንብልናሁ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ ለዓለ፡ ዓለ፡ ዓ፡ 
+                     <gap reason="omitted" unit="chars" quantity="3"/> ውእቱ፡ እግዚአብሔር፡ ስ<choice><sic>ብ</sic><corr>ቡ</corr></choice>ሕ፡ 
+                     ወጥቀ፡ ልዑል፡ ዲበ፡ <choice><sic>ኩ</sic><corr>ኵ</corr></choice>ሉ፡ ፍጥረት፡ ዝኢያገምሮ፡ ህሊና፡ ሰብእ፡ 
+                     ወኢአምሮ፡ ዘመላእክት፡ ለፈጣሬ፡ ሰማያት፡ ወምድር፡ እስመ፡ ውእቱ፡ አዘዘ፡ በቃለ፡ ጥበቡ፡ ወአቀሞሙ፡ ለዓለም፡ ትእዝዝ፡ ወሀቦሙ፡ ወኢሃለፉ፡ 
+                     <gap reason="omitted" unit="lines" quantity="10"/> ወለእለ፡ ይትለአከዎ፡ ነደ፡ እሳት፡ ስልስ፡ ዲቤ፡ ዝያ፡ ትአቦን፡ መልአከ፡ እግዚአብሔር፡ አውዶሙ፡ ለእለ፡
+                     ይሪፈርህዎ፡ ወያድኅኖሙ፡ ጠአሙ፡ ወታእምሩ፡ ከመ፡ ሄር፡ እግዚአብሔር፡ ወመንከር፡ ጥበብከ፡ እግዚኦ፡ ስብሐት፡ ለብ፡ ስብሐት፡ ለወልድ፡ ስብሐት፡ ለመንፈስ፡ ቅዱስ፡ ይዜኒ፡ 
+                     ወዘልፈኒ፡ ወለአለመ፡ ዓ፡ ዓ፡</ab>
+               </div>
+               
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="CommemorativeJoshua">
+                  <label xml:lang="en">Commemorative notice on Michael's appearance to Joshua</label>
```

Comment:
The following text is based on the account in `<bibl><ptr target="bm:2006DersanaMikael"/></bibl>`, but not on the manuscript record in BMLacq776.